packages/sth-config/src/config-service.ts

```diff
@@ -31,7 +31,8 @@ const _defaultConfig: STHConfiguration = {
         freeSpace: 128
     },
     safeOperationLimit: 512,
-    instanceAdapterExitDelay: 9000
+    instanceAdapterExitDelay: 9000,
+    runWithoutDocker: false
```

Comment:
_**Important:**_ Let's change this a lot actually...

As for this. In another PR (maybe just a PR to this PR after other fixes are done).

So the user should have a choice of adapters to use and should also be able to write and hook up own adapters. So the `get-{yournamehere}-adapter` class should include logic that would require any adapter the user would want... 

However... 

This does pose a security threat and a potential place to attack. So beside this should be: `allowedAdatptersRegexp: [/^@scramjet\//]` or something similar to this? Maybe `allowExtraAdapters`?